app/dao/notifications_dao.py

```diff
@@ -16,9 +18,24 @@ def get_notification_for_job(service_id, job_id, notification_id):
     return Notification.query.filter_by(service_id=service_id, job_id=job_id, id=notification_id).one()
 
 
-def get_notifications_for_job(service_id, job_id):
-    return Notification.query.filter_by(service_id=service_id, job_id=job_id).all()
+def get_notifications_for_job(service_id, job_id, page=1):
+    query = Notification.query.filter_by(service_id=service_id, job_id=job_id)\
+        .order_by(asc(Notification.created_at))\
```

Comment:
Chris and I chatted about this and went for newest first. We'll see it this makes sense to users in due course.